### PR TITLE
Corrected device class spelling

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: blueman/DeviceClass.py [hunk 1]
```diff
@@ -97,9 +97,9 @@
     # translators: device class
     _("Set-top box"),
     # translators: device class
-    _("Hifi audio"),
+    _("Hi-Fi audio"),
     # translators: device class
-    _("Vcr"),
+    _("VCR"),
```
Comment (comradekingu) [inline blueman/DeviceClass.py:102]: https://en.wikipedia.org/wiki/Videocassette_recorder

File: blueman/DeviceClass.py [hunk 2]
```diff
@@ -97,9 +97,9 @@
     # translators: device class
     _("Set-top box"),
     # translators: device class
-    _("Hifi audio"),
+    _("Hi-Fi audio"),
```
Comment (comradekingu) [inline blueman/DeviceClass.py:100]: https://en.wikipedia.org/wiki/High_fidelity

File: blueman/DeviceClass.py [hunk 3]
```diff
@@ -208,7 +208,7 @@
     # 19 - 48 reserved
     49: ('Pulse Oximeter', {0: _("Generic: Pulse Oximeter"),
                             1: _("Fingertip"),
-                            2: _("Wrist Worn")}),
+                            2: _("Wrist-Worn")}),
```
Comment (comradekingu) [inline blueman/DeviceClass.py:211]: https://dictionary.cambridge.org/us/dictionary/english/wrist

File: blueman/DeviceClass.py [hunk 4]
```diff
@@ -49,28 +49,28 @@
...
-    _("Isdn")
+    _("ISDN")
 )
 
 access_point_minor_cls = (
     # translators: device class
     _("Fully"),
     # translators: device class
-    _("1-17 percent"),
+    _("1–17 percent"),
```
Comment (comradekingu) [inline blueman/DeviceClass.py:63]: https://www.thepunctuationguide.com/en-dash.html

File: blueman/DeviceClass.py [hunk 5]
```diff
@@ -49,28 +49,28 @@
     # translators: device class
     _("Cordless"),
     # translators: device class
-    _("Smart phone"),
+    _("Smartphone"),
     # translators: device class
     _("Modem"),
     # translators: device class
-    _("Isdn")
+    _("ISDN")
```
Comment (comradekingu) [inline blueman/DeviceClass.py:56]: https://no.wikipedia.org/wiki/ISDN

File: blueman/DeviceClass.py [hunk 6]
```diff
@@ -49,28 +49,28 @@
     # translators: device class
     _("Cordless"),
     # translators: device class
-    _("Smart phone"),
+    _("Smartphone"),
```
Comment (comradekingu) [inline blueman/DeviceClass.py:52]: https://en.wikipedia.org/wiki/Smartphone

**Review by cschramm — APPROVED** [review]
Nice, thanks!

**Pull request merged**